### ISSUE TITLE
Program information cropped with overscan

### ISSUE THREAD

**Issue by yunosh** [hambre/Monochrome#3]
If the TV does overscanning (and you don't scale the video output, which would be a bad idea), the information OSD for a running program/recording gets cropped at the edge of the screen.

**Comment by hambre** [comment]
But this not a problem specific to this theme, is it? How does it work/look 
like on other themes?

**Comment by yunosh** [comment]
No, it's not. Themes that work well have have some border around or some padding inside the OSD.

![screenshot_20170126_223048](https://cloud.githubusercontent.com/assets/379318/22351208/22bcdf30-e417-11e6-8aeb-966ada581aa0.png)

**Comment by hambre** [comment]
All other windows of the theme are probably also affected. Almost all of them have information in a header bar (date, title, time) at the top of the screen and a footer bar at the bottom. Changing that means redesigning the whole theme, which I don't want to do.

**Comment by yunosh** [comment]
I don't consider this necessary. I have the MythTV UI rendered so that it compensates for the overscan. It fits exactly on the screen. But you cannot do this with video without e.g. losing the capability to use the TV's deinterlacing. Among other things. That's why the OSD is different.
